src/Sarif.Viewer.VisualStudio/Sarif.Viewer.VisualStudio.csproj

```diff
@@ -131,7 +132,6 @@
     <Compile Include="SdkUIUtilities.cs" />
     <Compile Include="ErrorList\SinkManager.cs" />
     <Compile Include="ErrorList\SarifTableDataSource.cs" />
-    <Compile Include="ErrorList\SarifSnapshot.cs" />
```

Comment:
>SarifSnapshot [](start = 32, length = 13)

Should this file be deleted from the repo?

Reply:
Yep. I'll take care of it.